build.sbt

```diff
@@ -26,3 +28,35 @@ lazy val toolkit = crossProject(JVMPlatform, JSPlatform, NativePlatform)
     ),
     mimaPreviousArtifacts := Set()
   )
+
+lazy val docs = project
+  .in(file("site"))
+  .enablePlugins(TypelevelSitePlugin)
+  .dependsOn(
+    toolkit.jvm // overkill? this way we can use mdoc to check the examples at least...
```

Comment:
@armanbilge before saying no, consider that in this way our examples can be typechecked by mdoc 😇